google61965e5d2c54f4a2.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google61965e5d2c54f4a2.html
```

Comment:
@sushyl: Do we need to add exception for this in .htaccess? I have a feeling that request for this file will be routed to index.php and will result in 404. Can you check on local?

Reply:
yeah, this might not work. let me try the meta-tags instead

Reply:
@dipakmdhrm added the metatag in the twig file instead.